.circleci/config.yml

```diff
@@ -797,6 +810,7 @@ workflows:
       - futures
       - boto
       - ddtracerun
+      - test_utils
```

Comment:
i think we need to add it as a requirement for wait_all_tests

Reply:
`wait_all_tests` does `tox -l` so it should get picked up.

That one seems automatic.

Reply:
nvm, see what you mean... this is hard